src/hexkit/providers/akafka/__init__.py

```diff
@@ -19,7 +19,10 @@
 respectively.
 """
 
-from .provider import KafkaEventPublisher  # noqa: F401
-from .provider import KafkaConfig, KafkaEventSubscriber
+from .provider import (
```

Comment:
This should be an absolute import. Can be fixed in future PR though, not important.

Reply:
I did not touch stuff that was already there and was not a direct issue with the move to ruff and pyproject.toml.
If we want to properly vet for relative imports, there are probably at least a few more places and it would make sense to have that change self-contained in its own PR.